db/migrate/20230126133440_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
You have an error when running your migrations this is the error I'm getting.
![Screenshot 2023-02-17 at 1 37 48 PM](https://user-images.githubusercontent.com/79506105/219644206-cbafef0e-99e4-4778-9b0a-4458ccb4cdf1.png)

![Screenshot 2023-02-17 at 1 37 09 PM](https://user-images.githubusercontent.com/79506105/219644528-b4c44b62-9794-4ca4-bbba-b92ea962f48b.png)

It would be nice if you fixed this issue.